src/App/App.js

```diff
@@ -31,18 +43,32 @@ class App extends Component {
                 <Link to='/'>
                   <h2>Home</h2>
                 </Link>
-                <Gallery cards={this.state.favorites} />
+                {!this.state.favorites.length &&
+                  <h2>You have not favorited any art yet</h2>}
+                {this.state.favorites.length &&
```

Comment:
<img width="416" alt="Screen Shot 2021-06-13 at 6 03 30 PM" src="https://user-images.githubusercontent.com/65516564/121825949-71a66380-cc72-11eb-8360-72dd59ded37a.png">
We've got a mysterious zero sneaking on the UI, and it looks like you could either use a true turnary or a double bang operator since `this.state.favorites.length` is just 0 and not a truthy value in the second context and rendering on the dom as a number.

Reply:
The same thing is happening on line 70 ^